lib/glob.js

```diff
@@ -1,10 +1,10 @@
-const bashGlob = require("bash-glob");
-const bashPath = require("bash-path");
+const globby = require("globby");
 
 module.exports = (...args) => {
-	if (!bashPath) {
-		throw new TypeError("`bash` must be installed"); // TODO move this check to bash-glob
-	}
+	const [pattern, ...options] = args;
 
-	return bashGlob.sync(...args);
+	return pattern.reduce(function (acc, p) {
+		acc = acc.concat(globby.sync(p, ...options));
+		return acc;
+	}, []);
```

Comment:
Oh, here is an issue: we should not invoke globby for each pattern.
We need to exec this cmd just once, and pass the all include/exclude patters.

Reply:
Done.